src/lib/keybindings.js

```diff
@@ -1,18 +1,18 @@
 const DEFAULT_BINDINGS = [
-  ['Select next header', 'selectNextVisibleHeader', 'ctrl+n'],
-  ['Select previous header', 'selectPreviousVisibleHeader', 'ctrl+p'],
+  ['Select next header', 'selectNextVisibleHeader', 'ctrl+down'],
+  ['Select previous header', 'selectPreviousVisibleHeader', 'ctrl+up'],
```

Comment:
I don't think "up" and "down" should require the `ctrl` modifier. But its also possible that this feature doesn't work without a modifier.